app/graphql/types/well_query_type.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Types
+  # The type for Well queries.
+  class WellQueryType < BaseObject
+    field :well, WellType, null: true do
+      description 'Find a Well by ID.'
+      argument :id, ID, required: true
+    end
+
+    def well(id:)
+      Well.find(id)
+    end
+
+    field :wells, [WellType], null: false do
+      description 'Find all wells, optionally those associated with a plate.'
+      argument :plate_id, Int, required: false
+    end
+
+    def wells(plate_id: nil)
+      wells = Well.all
```

Comment:
Well.find_by(plate_id: plate_id)

Reply:
Sorry. It looks like you are trying to find multiple wells so it should be Well.where(plate_id: plate_id) I think. The docs should tell you exactly how to do this.